libraries/rush-lib/src/logic/buildCache/ProjectBuildCache.ts

```diff
@@ -447,6 +448,18 @@ export class ProjectBuildCache {
     hash.update(RushConstants.hashDelimiter);
     hash.update(options.command);
     hash.update(RushConstants.hashDelimiter);
+    if (options.additionalContext) {
+      for (const key of Object.keys(options.additionalContext)) {
```

Comment:
```suggestion
      for (const key of Object.keys(options.additionalContext).sort()) {
```

Reply:
+1 for sort, it makes `"dependsOnEnvVars": ["FOO", "BAR"]` and `"dependsOnEnvVars": ["BAR", "FOO"]` produces the same hash